.travis.yml

```diff
@@ -11,13 +11,9 @@ python:
 addons:
   apt:
     packages:
-    - libmms0
+    - libmediainfo0
 matrix:
   include:
-    - python: 3.7
-      dist: xenial
-      # TODO: remove this hack once it's not required any more
-      sudo: true
```

Comment:
Have you tried using the `libmediainfo0v5` package? I don't want to drop Py3.7 tests. It should be available:
https://github.com/travis-ci/apt-package-safelist/blob/91531f827b40155b3de97b3b20db7c985c6e4c34/ubuntu-trusty#L7338